docs/howto-create-tenant.md

```diff
@@ -0,0 +1,170 @@
+5. Click **Directory role** and select **Global administrator** and then click **Ok** at the bottom of the pane.
+6. In the middle of the **User** pane, click **Show Password** and record the temporary password. After you log in the first time, you will be prompted to reset it.
+7. At the bottom of the pane, click **Create** to create the user.
+
+## Create an Azure Red Hat OpenShift cluster in your tenant
+
+Open a Bash command window and set the following variables:
+
+> [!IMPORTANT]
+> The name of your cluster must be all lowercase or cluster creation will fail.
+
+```BASH
+CLUSTER_NAME=<the name of your cluster - lowercase only>
+```
+
+ Use the same name for the cluster that you chose in step 6 of [Create new app registration](#create-new-app-registration).
+
+```BASH
+LOCATION=<location>
+```
+
+Choose a location to create your cluster, for example:
+`LOCATION=eastus`
+
+Set `FQDN` to the fully qualified name of your cluster. This is composed of the cluster name, the location, with `.cloudapp.azure.com` appended to the end.  For example:  
+
+```BASH
+FQDN=$CLUSTER_NAME.$LOCATION.cloudapp.azure.com
+```
+
+Set  `APPID` to the value you saved in step 1 of [Create a client secret](#create-a-client-secret).  
+```BASH
+APPID=<app ID value>
+```
+
+Set `SECRET` to the value you saved in step 7 of [Create a client secret](#create-a-client-secret).  
+```BASH
+SECRET=<secret value>
+```
+
+Set `TENANT` to the value you provided in step 7 of [Create a new tenant](#create-a-new-tenant)  
+```BASH
+TENANT=<tenant id>
+```
+
+Create the resource group for the cluster. Run the following command from the Bash shell that you used to defined the variables above:
+
+```BASH
+az group create --name $CLUSTER_NAME --location $LOCATION
+```
+
+Now create the cluster:
+
+```BASH
+az openshift create --resource-group $CLUSTER_NAME --name $CLUSTER_NAME -l $LOCATION --fqdn $FQDN --aad-client-app-id $APPID --aad-client-app-secret $SECRET --aad-tenant-id $TENANT
+```
+
+## Test your cluster
+
+To test your cluster you need to use a browser instance that hasn't cached the user you are currently logged into the Azure portal as. This is because you need to log in as the new user you created earlier.
+
+1. Open an incognito window (Chrome) or a InPrivate windows (Edge)
+2. Navigate to the sign-on URL that you created in step 6 of [Create new app registration](#create-new-app-registration). For example, https://mycluster.eastus.cloudapp.azure.com
+
+For now, you will get message that the site is not secure. Click **Details** > **Go on to the webpage**
+
+Log in with the user and password that you created in [Create new user in Active Directory](#create-new-user-in-active-directory)
+When the **Permissions requested** dialog appears, select **Consent on behalf of your organization** and then **Accept**.
+
+You should then see the Openshift container platform catalog in your browser. You now have an Azure Red Hat OpenShift cluster in your tenant.
+
+## Resources
+
```

Comment:
looks like these are not in new lines

Reply:
Fixed.  (An annoying markdown thing. Requires two spaces after those lines to generate a carriage return between each)